tests/samples/rename_git.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/added b/moved
+similarity index 85%
+rename from added
+rename to moved
+index a071991..4dbab21 100644
+--- a/added
++++ b/moved
```

Comment:
I see, your point is to add support for this specific git syntax.
My doubt resides in that this is git-specific and not unified diff standard syntax. Then the question would be if the lib should add support for particular VCS syntax...
Maybe I should take a look and evaluate how much work that would be. Will give it some thought.